cmd/run.go

```diff
@@ -73,6 +74,13 @@ func RunCmd(lg log.Logger, mt *metrics.StatsdMonitor, cfg config.Config) *cobra.
 			var failures = 0
 			report = append(report, []string{"Status", "Recipe", "Source", "Duration(ms)", "Records"})
 
+			bar := progressbar.NewOptions(len(recipes),
+				progressbar.OptionEnableColorCodes(true),
+				progressbar.OptionSetDescription("[cyan]running recipes [reset]"),
+				progressbar.OptionShowCount(),
+			)
+			//.Default(int64(len(recipes)))
```

Comment:
don't forget to remove this line

Reply:
oh sorry, forgot to save the file before commit.. 😅